frame/support/procedural/src/lib.rs

```diff
@@ -288,8 +288,8 @@ pub fn decl_storage(input: TokenStream) -> TokenStream {
 /// ```nocompile
 /// construct_runtime!(
 ///     pub enum Runtime where
-///         Block = Block,
-///         NodeBlock = runtime::Block,
+///         Block = runtime::Block,
+///         NodeBlock = Block,
```

Comment:
```suggestion
///         Block = Block,
///         NodeBlock = node::Block,
```

I think we should better do it that way, because `construct_runtime` is normally in the runtime and the block type is declared there.